web/src/main/java/io/openshift/appdev/missioncontrol/web/api/UploadForm.java

```diff
@@ -42,6 +42,10 @@
     @PartType(MediaType.APPLICATION_FORM_URLENCODED)
     private String openShiftCluster;
 
+    @FormParam("step")
+    @PartType(MediaType.APPLICATION_FORM_URLENCODED)
+    private String step;
```

Comment:
Can this be an Integer?

Reply:
That was my first idea, but I get:
```
RESTEASY007545: Unable to find a MessageBodyReader for media type: application/x-www-form-urlencoded and class type java.lang.Integer
```

Reply:
I wonder if it works using @HeaderParam instead